### PR TITLE
Update jp.m3u

### PR DESCRIPTION
**Pull request by malhuda**
Updated NHK

### CODE REVIEW

File: streams/jp.m3u [hunk 1]
```diff
@@ -186,9 +186,9 @@ http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (YouTube) (1080p)
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCSPEjw8F2nQDtmUKPFNF7_A/live
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (1080p)
```
Comment (UltraHDR) [inline streams/jp.m3u:188]:
```suggestion
Delete this line as it's a duplicate
```

File: streams/jp.m3u [hunk 2]
```diff
@@ -186,9 +186,9 @@ http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8
...
-https://nhkwlive-ojp.akamaized.net/hls/live/2003459/nhkwlive-ojp-en/index.m3u8
+https://nhkwlive-pre-in.webcdn.stream.ne.jp/hls/live/nhkwlive-pre/index_1M.m3u8
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (720p)
-https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp-en/index.m3u8
+https://nhkwlive-pre-in.webcdn.stream.ne.jp/hls/live/nhkwlive-pre/index_1M.m3u8
```
Comment (UltraHDR) [inline streams/jp.m3u:191]:
```suggestion
https://cdn.nhkworld.jp/www11/nhkworld-tv/global/2003458/live.m3u8
```
Replace with the adaptive link

File: streams/jp.m3u [hunk 3]
```diff
@@ -186,9 +186,9 @@ http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (YouTube) (1080p)
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCSPEjw8F2nQDtmUKPFNF7_A/live
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (1080p)
-https://nhkwlive-ojp.akamaized.net/hls/live/2003459/nhkwlive-ojp-en/index.m3u8
+https://nhkwlive-pre-in.webcdn.stream.ne.jp/hls/live/nhkwlive-pre/index_1M.m3u8
```
Comment (UltraHDR) [inline streams/jp.m3u:189]: Please delete this entry as it's a duplicate. It's also not 1080p

Comment (malhuda) [reply streams/jp.m3u:187]: Okay

**Review by Alstruit — APPROVED** [review]

File: streams/jp.m3u (hunk 3 above)
Comment (UltraHDR) [reply streams/jp.m3u:179]: It hasn't been deleted

File: streams/jp.m3u [hunk 4]
```diff
@@ -175,12 +175,10 @@ https://nhkw-zh-hlscomp.akamaized.net/ixxemlzk1vqvy44o/playlist.m3u8
 http://redlabmcdn.s.llnwi.net/nv02/ryowa1hd/index.m3u8
 #EXTINF:-1 tvg-id="NHKGeneralTV.jp" status="timeout",NHK G (480p) [Not 24/7]
 http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8
-#EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (YouTube) (1080p)
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCSPEjw8F2nQDtmUKPFNF7_A/live
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (1080p)
```
Comment (UltraHDR) [inline streams/jp.m3u:178]:
```suggestion
#EXTINF:-1 tvg-id="NHKWorldPremium.jp" status="timeout",NHK World Premium 720p)
```
https://github.com/iptv-org/iptv/issues/10955

File: streams/jp.m3u [hunk 5]
```diff
@@ -175,12 +175,10 @@ https://nhkw-zh-hlscomp.akamaized.net/ixxemlzk1vqvy44o/playlist.m3u8
 http://redlabmcdn.s.llnwi.net/nv02/ryowa1hd/index.m3u8
 #EXTINF:-1 tvg-id="NHKGeneralTV.jp" status="timeout",NHK G (480p) [Not 24/7]
 http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8
```
Comment (UltraHDR) [inline streams/jp.m3u:177]:
```suggestion
http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8
#EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (YouTube) (1080p)
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCSPEjw8F2nQDtmUKPFNF7_A/live
```

**Review by UltraHDR — CHANGES_REQUESTED** [review]

**Review by Alstruit — COMMENTED** [review]
Suggested changes.
```diff
diff --git a/streams/jp.m3u b/streams/jp.m3u
index a127eadb94..66470f1457 100644
--- a/streams/jp.m3u
+++ b/streams/jp.m3u
@@ -173,7 +173,9 @@ https://nhkw-zh-hlscomp.akamaized.net/ixxemlzk1vqvy44o/playlist.m3u8
 http://redlabmcdn.s.llnwi.net/nv02/ryowa1hd/index.m3u8
 #EXTINF:-1 tvg-id="NHKGeneralTV.jp" status="timeout",NHK G (480p) [Not 24/7]
 http://redlabmcdn.s.llnwi.net/nv02/ryowa2hd/index.m3u8
-#EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (1080p)
+#EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (YouTube) (1080p)
+https://hls.youtb.workers.dev/channel/UCSPEjw8F2nQDtmUKPFNF7_A.m3u8
+#EXTINF:-1 tvg-id="NHKWorldPremium.jp" status="timeout",NHK World Premium (720p)
 https://nhkwlive-pre-in.webcdn.stream.ne.jp/hls/live/nhkwlive-pre/index_1M.m3u8
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" status="timeout",NHK World Japan (720p)
 https://cdn.nhkworld.jp/www11/nhkworld-tv/global/2003458/live.m3u8

```

**Comment by LaneSh4d0w** [comment]
Closed due to lack of activity.

**Pull request closed without merging**